src/routes.js

```diff
@@ -0,0 +1,9 @@
+import Login from './components/Login';
+import Register from './components/Register';
+import Home from './components/Home';
+
+export default [
+    {path: '/login', component: Login},
```

Comment:
to avoid writing route path when we are calling route from components it's usually better to use route names. Each Route can have it's own unique name.